packages/core/src/components/Lifecycle/Lifecycle.tsx

```diff
@@ -20,6 +20,7 @@ import { makeStyles } from '@material-ui/core';
 
 type Props = CSS.Properties & {
   isShorthand?: boolean;
+  isAlpha?: boolean;
```

Comment:
Want to change these to just `shorthand` and `alpha` as well? prefixing with "is" shouldn't be done for boolean props